src/ApplicationLogs/LogReader.cs

```diff
@@ -68,6 +68,7 @@ public void OnPersist(StoreView snapshot, IReadOnlyList<Blockchain.ApplicationEx
                 JObject trigger = new JObject();
                 trigger["trigger"] = appExec.Trigger;
                 trigger["vmstate"] = appExec.VMState;
+                trigger["exception"] = appExec.Exception?.InnerException?.Message ?? appExec.Exception?.Message;
```

Comment:
Sometimes the `InnerException` has another `InnerException` please check https://github.com/neo-project/neo-node/blob/90c7f396df2ac52c4ced880c433d286719e578ce/neo-cli/CLI/MainService.cs#L560